pom.xml

```diff
@@ -189,6 +189,15 @@
 					<!-- Attach source jar to builds -->
 					<artifactId>maven-source-plugin</artifactId>
 					<version>${maven-source-plugin.version}</version>
+					<executions>
+						<execution>
+							<id>attach-sources</id>
+							<phase>verify</phase>
```

Comment:
```suggestion
```

The default is `package`, which should be sufficient.

see https://maven.apache.org/plugins/maven-source-plugin/jar-no-fork-mojo.html